src/ballet/sbpf/fd_sbpf_loader.c

```diff
@@ -499,33 +504,29 @@ _fd_sbpf_elf_peek( fd_sbpf_elf_info_t * info,
     .shndx_dyn        = -1,
     .shndx_dynstr     = -1,
     .phndx_dyn        = -1,
+    .sbpf_version     = 0U,
     /* !!! Keep this in sync with -Werror=missing-field-initializers */
   };
 
   fd_sbpf_elf_t const * elf = (fd_sbpf_elf_t const *)bin;
   int err;
 
   /* Validate file header */
-  if( FD_UNLIKELY( (err=fd_sbpf_check_ehdr( &elf->ehdr, elf_sz ))!=0 ) )
-    return err;
+  if( FD_UNLIKELY( (err=fd_sbpf_check_ehdr( &elf->ehdr, elf_sz, sbpf_max_version ))!=0 ) )
+    return NULL;
```

Comment:
I worry a little bit that we are throwing away the err information here and this change will throw a bunch of fuzz errors that we will have to fix later... Was the returned err masked out in the caller?

Reply:
Agreed. We can bring there errors back again if needed. I think we're already fuzzing elf loader, so prob good for now. 🤞